source/lib-intro.tex

```diff
@@ -219,6 +225,20 @@
 direct-initialization\iref{dcl.init}
 that is not list-initialization\iref{dcl.init.list}
 
+\definition{expression-equivalent}{defns.expression-equivalent}
+\indexdefn{expression-equivalent}%
+relationship that exists between two expressions \tcode{E1} and \tcode{E2} such that
+\begin{itemize}
```

Comment:
ISO wants us to phrase definitions of such terms in a way so that you can macro-expand the definition into the point-of-use and the sentence stays intact (modulo semantic parentheses implied by the macro). I think this wouldn't work here.

Reply:
I'll fix this.